spring-rabbit/src/main/java/org/springframework/amqp/rabbit/log4j2/AmqpAppender.java

```diff
@@ -605,6 +609,11 @@ public int incrementRetries() {
 		 */
 		private String charset = Charset.defaultCharset().name();
 
+		/**
+		 * Whether or not add MDC properties into message headers.
+		 */
+		private boolean addMdcAsHeaders = false;
```

Comment:
Why false? Originally it was `true` for backwards compatibility.

Reply:
Hey, yes, right, it was. I changed my mind by some reasons:
this modification will be included into one of next versions. if someone will update packages and will find missing data (MDC) in logs it is obviosly will be found very soon and it will be clear that it is related to the packages updating. it is not very good but I think it is ok - that person can will check documentation and will find a solution.
And an opposite, if we will keep it as before then people will miss some messages in logs completely while most of messages will be present. It will less observable, can be annoying and they will need much more time to find a solution.

Reply:
I'm fine having it as a `false` by default in the current version. 
Doesn't look like we are going to back-port it...

Reply:
I think it should be `true`; I don't think most people store large data in MDC.

Reply:
Oh! I didn't look at that like this.
So, fine with me either way.

@egusev, your word is last 😄

Reply:
sorry guys, my word could be only a first one :)
the main point from my point of view is that logs miss happens unexpectedly and requires time to investigate. having false by default we  eliminate the problem for futher usages. but `true` is also fine for me